src/main/webapp/i18n/en/featureOverview.json

```
@@ -165,7 +165,7 @@
                 "onlineReview": {
                     "title": "Online Exam Review",
                     "shortDescription": "Review the assessment of your exam from anywhere in the world.",
-                    "descriptionTextOne": "After the results have been published, you can access your exam and review the assessment comfortably from your computer. Take your time to really analyze the question and the assessment to understand what mistakes you made. There are no location restrictions to the exam review, you can access it from anywhere! \n\nSometimes mistakes can happen during the exam assessment. Using the complaint functionality, you can complain about an assessment and provide some information on why you feel you were not assessed correctly. All student complaints will be collected and a different assessor will have the chance to either accept or reject your complaint. If your complaint is accepted, your points will be automatically updated!"
+                    "descriptionTextOne": "After the results have been published, you can access your exam and review the assessment comfortably from your computer. Take your time to really analyze the question and the assessment to understand what mistakes you made. There are no location restrictions to the exam review, you can access it from anywhere! \n\nSometimes mistakes can happen during the exam assessment. Using the complaint functionality, you can complain about an assessment and provide some information on why you feel you were not assessed correctly. All student complaints will be collected, and a different assessor will have the chance to either accept or reject your complaint. If your complaint is accepted, your points will be automatically updated!"
```

```suggestion
                    "descriptionTextOne": "After the results have been published, you can access your exam and review the assessment comfortably from your computer. Take your time to really analyze the question and the assessment to understand what mistakes you made. There are no location restrictions to the exam review, you can access it from anywhere! \n\nSometimes mistakes can happen during the exam assessment. Using the complaint functionality, you can complain about an assessment and provide some information on why you feel you were not assessed correctly. All student complaints will be collected and a different assessor will have the chance to either accept or reject your complaint. If your complaint is accepted, your points will be automatically updated!"
```
nitpick: I would not have placed a comma here.

Can be left in since the rule for commas in the linked LanguageTool doc mentions that it is not a hard rule if the comma should be there or not but may be up to preference in some cases.